worldeditadditions_commands/commands/selectors/scol.lua

```diff
@@ -0,0 +1,42 @@
+-- ███████  ██████  ██████  ██
+-- ██      ██      ██    ██ ██
+-- ███████ ██      ██    ██ ██
+--      ██ ██      ██    ██ ██
+-- ███████  ██████  ██████  ███████
+worldedit.register_command("scol", {
+	params = "[<axis1>] <length>",
+	description = "Set WorldEdit region position 2 at a set distance along 1 axis.",
+	privs = {worldedit=true},
+	require_pos = 1,
+	parse = function(params_text)
+		local wea, vec, tmp = worldeditadditions, vector.new(0, 0, 0), {}
```

Comment:
The `tmp` table here is odd. What is it's purpose? Perhaps it could be renamed to better reflect it's purpose.

Also, doing a `local wea = worldeditadditions` at the top of the file would mean you'd have access to it for the entire file, instead of just in the `parse` function here - but this isn't a high-priority thing to change here :-)

Reply:
Good spot on the `wea` declaration... I'll implement that. As for `tmp` it's just a dump for all the misc data that I don't want to contaminate `vec` with (although now that I think about it I could have just made the vector table a sub-table of `vec`... but that's too complicated).

Reply:
Reimplemented wea outside of the parse function.